src/client/fdsn.py

```diff
@@ -53,7 +53,10 @@
 
 g_default_site = 'geofon'
 
-g_timeout = 20.
+if config.config().fdsn_timeout is None:
```

Comment:
is this necessary with `Config.fdsn_timeout.T(default=20.)`?

Reply:
no, but was requested by @emolch to be be None by default